packages/micropip/test_micropip.py

```diff
@@ -186,7 +186,29 @@ def test_install_different_version(selenium_standalone_micropip):
     )
     selenium.run_js(
         """
-        pyodide.runPython(`
+        await pyodide.runPythonAsync(`
+            import pytz
+            assert pytz.__version__ == "2020.5"
+        `);
+        """
+    )
+
+
+def test_install_different_version2(selenium_standalone_micropip):
```

Comment:
It might be worth adding a test that would check most of it outside of pyodide, similarly to `test_add_requirement_relative_url`. Given that `selenium_standalone` fixture is quite expensive time wise, also it's not ideal if we have to make calls to PyPi for every single test in micropip.

Reply:
We could get rid of `selenium_standalone` by deleting the module from `sys.modules`, deleting the package from `site-packages`, and removing the information from `pyodide.loadedPackages` and `micropip.installed_packages`. I could try to make a fixture that does this and use it in all tests that use `selenium_standalone` for package-loading reasons.

(Also we should probably merge `installed_packages` and `loadedPackages`.)

Reply:
In this particular case, removing `sys.modules` is not enough I think as there could be potentially wrong downloaded files in the filesystem anyway. Also I'm not sure the extra complexity is worth it, as it would make it harder to debug when it fails.

I was thinking just checking the resulting transaction objects and assuming that the part that processes those transactions works correct. Though it's true that an integration tests with `selenium_standalone` wouldn't hurt which puts us back to the starting point. We can also keep it as it is now.

Reply:
> We can also keep it as it is now.

Sounds good. I can remove the test that imports the exact wheel so that there only one integration test for this feature rather than two.